src/packages/@ncigdc/components/Header.js

```diff
@@ -22,10 +22,15 @@ import { setModal } from "@ncigdc/dux/modal";
 import SessionExpiredModal from "@ncigdc/components/Modals/SessionExpiredModal";
 
 import Banner from "@ncigdc/uikit/Banner";
+import activeTheme from "@ncigdc/theme/versions/active";
```

Comment:
Those are great ideas, I would approve that PR
In the meantime I went with hopefully a path of less resistance